CMSgov/easi-app
# [Easi-1905] Create "Templates for 508 testing" help article

# EASI-1905

## Changes and Description

- Created new `TestingTemplateBase` for new article 
- Tweaked existing '508 Templates' view/component
- Tweaked existing component to reuse any area within EASI
- Added tests



## How to test this change

1. Navigate to http://localhost:3000/help
2. Find and click on the link:  `View Section 508 articles`
3. Find and click on the link:  `Templates for 508 testing`
4. Ensure new tab opens with the article.
5. Ensure article look and copy reflects figma

## JIRA Acceptance Criteria

- Reuse existing '508 Templates' view/component
- May need to tweak existing component to be reusable in any area within EASI
- Testing where applicable
